plan.md

```diff
@@ -1,9 +1,282 @@
++ Improved focussed time
++ Effective time management (understand where your time is going & how to improve)
++ Improved team communication
++ Focus on the individual's potential/productivity first
++ Ability to share with the team built in from the beginning so people know
+when they would be interrupting ("I'm on fire" one-click warning to others
+:wink:) - key is to do a lot of research around what people _want_ to share
+and what they want to be shared _with them_
+
+#### CRM
+We have outgrown our 'spreadsheet' customer relationship management (CRM) system
+and need a place where we can **securely and accessibly** store all our client
+communications so that the *whole team* can see and contribute to these.
+
+Our CRM *code* will be **fully open source**, following our [guiding principles](https://github.com/dwyl/phase-two#guiding-principles) and
+removing as many barriers to entry/use as possible for small businesses with a
+social impact. The data however, will not be accessible to anyone outside of the
+organisation or sold to 3rd parties (a la [LinkedIn](https://www.forbes.com/sites/georgeanders/2012/06/27/how-linkedin-strategy/#c4e2c1fdf82c)).
+
+Value lies in:
++ Linking all of the branches of our business together
++ Excellent low risk learning project for learning our new stack
++ Modular architecture which is easy to extend *without* becoming super patchy
+  + Modules re-used across multiple products and services projects
++ Transparency on business development for all of our team
++ Central location where data can easily be entered as our business development
+efforts expand with [Services](#services) and [Training](#training) offerings
++ 50% of all charitable organisations we have spoken to have told us that their
+current CRM is not fit for purpose, we could easily begin to provide a solution
+to this problem if we desired
+
+> There are many 'small' enhancements we will be making to increase the effectiveness
+of our own workflow. **Each should be evaluated as a potential product.**
+
+
+### Services
+
+Our Services arm is how we help other organisations serve _their_ communities and achieve their social goals.
+
+We *already* do far more than just 'web development' for our services clients
+and this should be formalised into our offering.
+
+#### ATM Workshops  
+Deeply understanding the problem and helping organisations to arrive at a
+solution with actionable metrics is where dwyl adds the most value.
+
+Our 1 to 5 day workshops will be the cornerstone of the Services offering,
+providing clients with:
++ Understanding the "success drivers" i.e. "ATMs"
++ Proposed technical path forward - what technology can the client use to be more effective at reaching their organisational goals
++ Simple prototype
++ Effective [agile collaboration and user testing training](#training)
+
+These workshops will be our largest 'per man hour' contributor to fixed overheads
+from our Services branch by the end of 2017.
```

Comment:
@markwilliamfirth Agree, if we use this wording, we'll have to expand to include the answer to that question 👍 

The answer is what's stated in the text, but could be reformulated to avoid negative perceptions :blush: